app/views/courses/show.html

```diff
@@ -0,0 +1,107 @@
+<div class='courses-show-container' ng-controller='CoursesShowCtrl as controller'>
+  <!-- Page Heading -->
+  <div class='page-header break-words'>
+    <h1 class='inline'>
+      <span translate='i18nCourses'></span>
+    </h1>
+    <h2 class='inline'>
+      <span> - </span>
+      <span> {{controller.course.name}}</span>
+      <small> - </small>
+      <small translate='i18nInformation'></small>
+    </h2>
+  </div>
+  <!-- /Page Heading -->
+
+  <!-- Course Data -->
+  <div class='data-container'>
+    <div class='data-item'>
+      <p class='bold static-label' translate='i18nCourseId'></p>
+      <p class='break-words'>{{controller.course.courseId}}</p>
+    </div>
+    <div class='data-item'>
+      <p class='bold static-label' translate='i18nCourseName'></p>
+      <p class='break-words'>{{controller.course.name}}</p>
+    </div>
+    <div class='data-item'>
+      <p class='bold static-label' translate='i18nCredits'></p>
+      <p class='break-words'>{{controller.course.credits}}</p>
+    </div>
+    <div class='data-item'>
+      <p class='bold static-label' translate='i18nSemester'></p>
+      <p class='break-words'>{{controller.course.semester}}</p>
+    </div>
+  </div>
+  <!-- /Course Data -->
+
+  <!-- Correlatives -->
+  <h3 translate='i18nCorrelatives' class='page-header'></h3>
```

Comment:
I would apply `page-header` class to this item, but yes a similar style.
I would create a class `page-sub-header` with the same style as the page header, and additionally, a `padding-top: 15px` so as to separate the previous content of the current one.